.github/workflows/ci.yaml

```diff
@@ -532,7 +532,10 @@ jobs:
         with:
           container-runtime: ${{matrix.container-runtime}}
       - run: minikube image load /tmp/test.tar
-      - name: cargo test containerd
+      # By running the test of the targetless agent first, we prove it works on an empty cluster without any pods.
+      - name: Run targetless E2E test.
+        run: cargo test -p tests targetless
+      - name: Run all E2E test
```

Comment:
`targetless` will run twice :<

Reply:
Yeah, once on an empty cluster, and once on a cluster with pods. This is good. The test should not take a long time either.